_drafts/2017-07-27-taxonomy-suite.md

```diff
@@ -209,9 +210,9 @@ tbl(src, sql("select * from taxonomic_units limit 10"))
 
 `taxa` is our newest entry (hit CRAN just a few weeks ago) into the taxonomic R package space. It defines taxonomic classes for R, and basic, but powerful manipulations on those classes.
 
-It defines two broad types of classes: those with just taxonomic names data, and those with taxonomic names data plus other data (such as traits, environmental data, etc.) - so called `taxmap` classes.
+It defines two broad types of classes: those with just taxonomic data, and a class with taxonomic data plus other associated data (such as traits, environmental data, etc.) called `taxmap`.
 
-With these taxonomic classes, you can do various operations on them. With the taxonomic classes, you can filter out or keep taxa based on various criteria, and with the `taxmap` classes when you filter on taxa, the associated data is filtered the same way so taxa and data are in sync.
+The `taxa` package includes function to do various operations with these taxonomic classes. With the taxonomic classes, you can filter out or keep taxa based on various criteria. In the case of the `taxmap` class, when you filter on taxa, the associated data is filtered the same way so taxa and data are in sync.
 
 A manuscript about `taxa` is being prepared at the moment - so look out for that.
 
```

Comment:
Text "A quick example of the power of taxa" really needs to say up front what the example will do. What's the power that you're bragging about? ;-)